doc/generic/pgf/text-en/pgfmanual-en-tikz-matrices.tex

```diff
@@ -477,33 +477,33 @@ \subsubsection{Cell Styles and Options}
 
 In some cases, it is desirable to include some automation in each column/row
 separately. A typical example is to apply stripe-pattern to almost all columns
-and treat only a few of them differently. Then, nesting these keys can open up
+with exceptions. For these type of use-cases, nesting these keys can open up
 a lot of possibilities; in the following example a ``feature comparison'' table
-is demonstrated. It is intentionally made rather verbose to show how the column
-and row settings can be progressively overwritten to create certain effects.
-
-\begin{codeexample}[preamble={\usetikzlibrary{matrix,shapes.misc}}]
-\begin{tikzpicture}
-  [mycol/.style={column #1/.append style={
-     every even row/.style={nodes={fill=gray!10}}}
-   }]
+is demonstrated. It is intentionally made rather verbose and a bit redundant
+to show how the column and row settings can be progressively overwritten to
+create certain effects.
+
+\begin{codeexample}[preamble={\usetikzlibrary{matrix,fit}}]
```

Comment:
```suggestion
{
    \definecolor{ocher}{RGB}{230,230,150}
\begin{codeexample}[preamble={\usetikzlibrary{matrix,fit}}]
```

Reply:
can we find a mix of known colors so that we skip this definition? It is a bit crowded already that's why I'm trying to run away from definitions